tests/test_wizard_postgres.py

```diff
@@ -87,3 +95,84 @@ def test_postgres():
     assert 'title' in str(ref_list)
     assert 'url' in str(ref_list)
     assert 'research-hub.nrel.gov' in str(ref_list)
+
+
+def test_ref_replace(mocker):
+    """Test to ensure removal of double quotes from references."""
+    mock_conn_cm = mocker.MagicMock()
+    mock_conn = mock_conn_cm.__enter__.return_value
+    mock_conn.cursor.return_value = Cursor()
+
+    mock_connect = mocker.patch('psycopg2.connect')
+    mock_connect.return_value = mock_conn_cm
+
+    wizard = EnergyWizardPostgres(db_host='Dummy', db_port='Dummy',
+                                  db_name='Dummy', db_schema='Dummy',
+                                  db_table='Dummy',
+                                  boto_client=BotoClient(),
+                                  meta_columns=['title', 'url', 'id'])
+
+    refs = [(chr(34), 'test.com', '5a'),
+            ('remove "double" quotes', 'test_2.com', '7b')]
+
+    ids = np.array(['7b', '5a'])
+
+    out = wizard._format_refs(refs, ids)
+
+    assert len(out) > 1
+    for i in out:
+        assert json.loads(i)
```

Comment:
Hmmm, this would never actually fail, right? since you call `json.dumps` in the last line of the `_format_refs` method, the output should always be loadable (if not, that's on the `json` library to fix, not yours). I would recommend checking the content of the dictionaries that are loaded by `json.loads`, but I guess I won't insist - you can make the call!

Reply:
Ok I went ahead and changed this one, but does the fact that I call json.dumps in the function make this test a little redundant?

Reply:
I don't think so. Even if your only goal with the replacements was to fix errors during `json.dump`, you still want a test to make sure that any future changes don't regress (i.e. that `chr(34)` is always replaced, even as the code lives on). Without a test, you wouldn't know if this particular case still works or not.

On top of that, this test is a good setup for testing future replacements that you may need that have nothing to do with `json.dump` (e.g. maybe in the future you always want to replace lowercase "nrel" with uppercase "NREL").